addons/docs/src/blocks/Story.tsx

```diff
@@ -71,14 +73,15 @@ export const getStoryProps = (
       `Story '${storyName}' is set to render inline, but no 'prepareForInline' function is implemented in your docs configuration!`
     );
   }
-
-  return {
+  const newVar = {
```

Comment:
what is `newVar`?

Reply:
After adding the `width` property to the StoryProps interface, TypeScript kept giving me the following error:
![image](https://user-images.githubusercontent.com/48736073/70853349-fb048c00-1e69-11ea-8192-4b29cc4939a2.png)

I kept getting the error whenever I would return the object like this:
![image](https://user-images.githubusercontent.com/48736073/70853367-5b93c900-1e6a-11ea-9634-c5ff7f8c0807.png)

After inserting the object into a constant and then returning it, the error cleared out.
![image](https://user-images.githubusercontent.com/48736073/70853464-a82bd400-1e6b-11ea-8df3-3bde247ce26e.png)